core/include/core/G3Constants.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _G3_CONSTANTS_H
+#define _G3_CONSTANTS_H
+
+#include <cmath>
+#include <G3Units.h>
+
+namespace G3Constants {
+	using namespace G3Units;
```

Comment:
Does this end up making G3Constants::K work by accident?

Reply:
Hmm possibly, haven't checked.  Is there a more correct way to do this, other than having to explicitly spell out `G3Units::` everywhere?

Reply:
After a quick test, it does make G3Constants::K work. I am not aware of an alternative except spelling out G3Units::.

Reply:
Okie doke